subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/api/DeprecatedConfigurationsIntegrationTest.groovy

```diff
@@ -119,12 +116,9 @@ class DeprecatedConfigurationsIntegrationTest extends AbstractIntegrationSpec {
         """
 
         when:
-        executer.expectDocumentedDeprecationWarning("The compileOnly configuration has been deprecated for resolution. This will fail with an error in Gradle 8.0. " +
-            "Please resolve the compileClasspath configuration instead. " +
-            "Consult the upgrading guide for further information: https://docs.gradle.org/current/userguide/upgrading_version_5.html#dependencies_should_no_longer_be_declared_using_the_compile_and_runtime_configurations")
+        fails 'resolve'
 
         then:
-        succeeds 'resolve'
+        failure.hasErrorOutput("Dependencies can no longer be declared using the `compileOnly` configuration.")
```

Comment:
This looks like the wrong error?

It's supposed to be about using compileOnly and resolving it?